datafusion/proto/tests/cases/roundtrip_physical_plan.rs

```diff
@@ -946,10 +946,22 @@ fn roundtrip_scalar_udf_extension_codec() -> Result<()> {
         )),
         input,
     )?);
+    let udaf = AggregateUDF::from(Max::new());
+    let aggr_expr = create_aggregate_expr(
+        &udaf,
```

Comment:
@edmondop You should change to `max_udaf` here. I test it locally and the test pass #11595 might not be an issue actually cc @Lordworms